text/0000-fix-fees.md

```diff
@@ -0,0 +1,100 @@
+The already present accept\_fee\_base parameter appears suitable for this, as
+there is no reason to use different fees for relay and mempool acceptance. Its
+value shall default to GRIN\_BASE / 100 / 20 = 500000, which make each output
+incur just over 1 Grin-cent in fees.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+I think there is only a perceived drawback, which is what led to the former fee rules.
+A negative weight on inputs is supposed to encourage spending many outputs, and lead to a reduction in UTXO size.
+A positive weight on inputs is not really a drawback however, as long as creation of outputs is more strongly discouraged.
+Most wallets, especially those relying on payjoins, do about equal amounts of output creation and spending,
+the difference being just the wallet's UTXO set. 
+Mining pools are the major exception. While they obviously don't incur any fees in coinbase creation, they need not
+spend fees on spending them either, as they can mine these payout transactions directly.
+
+# Rationale and alternatives
+[rationale-and-alternatives]: #rationale-and-alternatives
+
+There are no good alternative to economically significant fees. Any blockchain lacking them is an open invitation to abuse.
+For chains with a maximum blocksize, fees are also necessary to allow prioritization of transactions.
+
+
+# Prior art
+Several chains have suffered spam attacks. In early days, bitcoin was swamped with feeless wagers on Satoshi Dice [1].
+At least those served some purpose.
+
+Nano was under a deluge of meaningless 0-value back and forth transfers for weeks,
+that added dozens of GB of redundant bloat to its chain data [2]. Although nano requires client PoW as a substitute for fees,
+these attacks showed that the PoW was poorly calibrated and not an effective deterrant.
+
+
+# Unresolved questions
+[unresolved-questions]: #unresolved-questions
+
+# Future possibilities
+[future-possibilities]: #future-possibilities
+
+While the input and kernel weights are pretty fixed, the output weight is subject to improvements in range proof technology.
+If Grin implements BP+, the weight should go down from 21 to 18. That would make outputs incur slightly less than a Grin-cent in fees,
+which is not worth bothering with. If range proofs were to halve in size though, then we might want to double the base fee to compensate.
+
+In Grin ever becomes worth many dollars, then a lowering of fees is desirable.
+This can then be achieved by getting the majority of running nodes to reconfigure their base fee to a lower value,
+after which wallets can have their fee computation adjusted.
+In the converse case, where Grin becomes worth only a few cents, then an increase in fees might be needed to avoid spam.
+Both cases will be much easier to deal with if they coincide with a hard fork, but those may not be on the horizon.
+
+
+# References
+[references]: #references
+
+[1] https://en.bitcoin.it/wiki/Satoshi\_Dice
```

Comment:
the link does not work correctly

Reply:
I put a backslash before the _ to appease my markdown editor. Big mistake:(